web-frontend/src/mirage/factories/account.ts

```diff
@@ -0,0 +1,26 @@
+import { Factory } from 'miragejs';
+import { faker } from '@faker-js/faker';
+import { AccountObject } from '../../models/AccountModel';
+
+export const accountFactory = Factory.extend<AccountObject>({
+  nickname() {
+    return faker.internet.userName();
+  },
+  candidateBalance() {
+    return faker.number.int().toString();
+  },
+  balance() {
+    return faker.number.int().toString();
+  },
+  address() {
+    return 'AU' + faker.string.alpha({ length: { min: 128, max: 256 } });
```

Comment:
not blocking but if I am not wrong an address is 53 char max

Reply:
No! Address can be long but we can check that with @Thykof  🤔

Reply:
for the sake of the faker, min 50 max 53